src/translators/crate.py

```diff
@@ -42,13 +42,15 @@
 class CrateTranslator(sql_translator.SQLTranslator):
     NGSI_TO_SQL = NGSI_TO_SQL
 
-    def __init__(self, host, port=4200, db_name="ngsi-tsdb"):
+    def __init__(self, host, port=4200, db_name="ngsi-tsdb", username=None, password=None):
```

Comment:
i don't recall where, but i experienced the case that i had to put 'crate' as username if using the driver with username to connect. if that the case, maybe better 'crate' as default value.

Reply:
as far as i know (just looking in my memory, not in the docs) crate uses "crate" as username per default if the connection is done via the crate driver. if the connection uses the postgres driver "crate" needs to be used as username explicitly.

As the tests run fine with the (non-auth) crate container i think it works well.

i can set the default to crate instead of None and run the tests again, but i do not think that it'll make any difference